cmd/minishift/cmd/start.go

```diff
@@ -362,9 +360,9 @@ func determineIsoUrl(iso string) string {
 	iso = strings.ToLower(iso)
 
 	switch iso {
-	case b2dIsoAlias, "":
+	case startFlags.B2dIsoAlias, "":
```

Comment:
in this case we might also be able to use a internal variable to make `""` refer to a named option such as `DefaultIsoOption` to make it clearer why we have this in the case select.